doc/source/_toc.yml

```diff
@@ -193,6 +193,7 @@ parts:
               - file: serve/deploy-many-models/multi-app
               - file: serve/deploy-many-models/model-multiplexing
           - file: serve/configure-serve-deployment
+          - file: serve/grpc-guide
```

Comment:
Let's have this in advanced guides since it is still alpha. It should replace the existing guide